docs/stratis.txt

```diff
@@ -149,10 +149,14 @@ Mailing list::
 	stratis-devel@lists.fedorahosted.org for general development
 	discussion
 Unknown values::
-	stratis-cli uses the Stratis API. If the API returns values that
-	stratis-cli cannot interpret, stratis-cli will substitute "???". If
-	encountered, upgrading to the latest version of stratis-cli, or filing
-	an issue, is recommended.
+	If the stratisd D-Bus API returns values that stratis-cli cannot
+	interpret, stratis-cli will substitute "???". If encountered,
+	upgrading to the latest version of stratis-cli, or filing an
+	issue, is recommended.
+Unobtainable values::
+	If the stratisd D-Bus API detects that a value is unobtainable,
```

Comment:
^detects^indicates..the D-Bus API layer is really just passing information through from the engine, so it should not really detect anything.

Reply:
Ah, okay; updated.